doc/cmd/scan/supported_resources/aws.md

```diff
@@ -142,7 +142,9 @@ As AWS documentation recommends, the below policy is granting only the permissio
                 "dynamodb:ListTagsOfResource",
                 "dynamodb:DescribeTimeToLive",
                 "dynamodb:DescribeTableReplicaAutoScaling",
-                "dynamodb:DescribeContinuousBackups"
+                "dynamodb:DescribeContinuousBackups",
+                "route53:ListHealthChecks",
+                "route53:GetHealthCheck"
```

Comment:
Can we try to keep these permissions in order and put the two new route53 policies just next to the others. Thanks :)

Reply:
Ok I'll group them by service